explainer.md

```diff
@@ -29,46 +29,35 @@ The APIs in scope of this group will not be tied to any particular platform and
 
 Depending on the underlying hardware capabilities, these platform APIs may make use of CPU parallelism, general-purpose GPU, or dedicated ML hardware accelerators. The API will provide high-level hints to web developers to enable [performance adaptation](https://webmachinelearning.github.io/webnn/#usecase-perf-adapt), but will remain hardware agnostic otherwise.
 
-## Getting started / example code
+## Getting started
 
-Provide a terse example for the most common use case of the feature.  If you need to show how to get the feature set up (initialized, or using permissions, etc.), include that too
+>Provide a terse example for the most common use case of the feature.  If you need to show how to get the feature set up (initialized, or using permissions, etc.), include that too.
 
-[here]
+A core abstraction behind popular deep learning frameworks is a computational graph. A computational graph is a directed graph with its nodes corresponding to operations (ops) and input variables. One node's output value is the input to another node. WebNN API brings this abstraction to the Web.
 
-## Key scenarios
-
-Next, discuss the key scenarios which move beyond the most canonical example, showing how they are addressed using example code:
-
-### Scenario 1
-
-Outline the scenario, then provide:
-
-[sample code that demonstrates the feature]
+In WebNN API [`Operand`](https://webmachinelearning.github.io/webnn/#operand) objects represent input, output, and constant tensors. [`NeuralNetworkContext`](https://webmachinelearning.github.io/webnn/#api-neuralnetworkcontext) defines a set of computationally-intensive ops that can be accelerated with dedicated hardware and are required by [models](https://github.com/webmachinelearning/webnn/blob/master/op_compatibility/first_wave_models.md) that address identified [use cases](https://webmachinelearning.github.io/webnn/#usecases). In addition, WebNN API provides affordances to build a computational graph, compile the graph, and execute the graph.
```

Comment:
- Might want to also explain what 'tensors' are e.g. "..input, output, and constant multi-dimensional arrays known as 'tensors'..."

- Not all operations in the computational graph are computationally-intensive, most are just there to massage the data e.g. "...defines a set of operations that may be accelerated with dedicated hardware such as the GPUs or dedicated AI accelerators, ..."

- May want to also mention that WebNN API affords integrating the computational graph with the rest of the Web media API e.g. passing image or video frames as inputs to WebNN graph.

Reply:
Thanks! Great comments. I updated the PR. PTAL.